examples/b2c/checkout/integrations/classes/B2BDeliverySample.cls

```diff
@@ -12,6 +12,9 @@ global class B2BDeliverySample implements sfdc_checkout.CartShippingCharges {
             // We're getting information like rates and carriers from this external service. 
             String siteLanguage = jobInfo.siteLanguage; 
 
+            // Following line of code calls the live HTTP Heroku Service to retrieve Shipping Methods.
```

Comment:
// Following line of code calls the live HTTP Heroku Service to retrieve mocked sample shipping methods.